package.json

```diff
@@ -118,7 +120,7 @@
       }
     },
     "test": {
-      "command": "npm run version-check && npm run unittest && npm run servertest && npm run eslint && npm run stylelint",
+      "command": "npm run version-check && npm run unittest && npm run servertest && npm run flow:check && npm run eslint && npm run stylelint",
```

Comment:
While I usually like running linters after "real" tests, I wonder if running flow first might help us find type errors in our code (exactly what flow can be useful for). Maybe we want to run flow before other tests?